### ISSUE TITLE
Touch sensor

### ISSUE THREAD

**Issue by noelbjohn** [ShapeLab/SwarmUI#16]
How to make the touch sensor circuit. Or is it available at any online electronic stores?

**Comment by Jharilela** [comment]
Yes, I need help with this as well, the pattern of the circuit is provided on [SwarmUI/Hardware/3D Model/v1/Touch Circuit.pdf](https://github.com/ShapeLab/SwarmUI/blob/master/Hardware/3D%20Model/v1/Touch%20Circuit.pdf)  but there is no instruction as which material to be used and how to print the pattern on the material

**Comment by PatrickGoettsch** [comment]
Did anyone find a product?

**Comment by mathieulegoc** [comment]
Any flexible PCB material would work, we used Dupont Pyralux (https://www.adafruit.com/product/1894) that we chemically etched after printing the pattern with a standard laser printer.

**Comment by PatrickGoettsch** [comment]
Thank you very much. What is the reason for etching the pattern? Weight reduction?

 

Von: Mathieu Le Goc [mailto:notifications@github.com] 
Gesendet: Mittwoch, 15. November 2017 20:56
An: ShapeLab/SwarmUI <SwarmUI@noreply.github.com>
Cc: Patrick Göttsch <Patrick.Goettsch@tuhh.de>; Comment <comment@noreply.github.com>
Betreff: Re: [ShapeLab/SwarmUI] Touch sensor (#16)

 

Any flexible PCB material would work, we used Dupont Pyralux (https://www.adafruit.com/product/1894) that we chemically etched after printing the pattern with a standard laser printer.
